patterns/stories-podcast-hero.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"metadata":{"categories":["featured","hero"],"patternName":"twentytwentyfive/stories-podcast-hero","name":"Stories Podcast Hero"},"align":"full","className":"has-accent-2-background-color has-background","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}},"backgroundColor":"accent-2"},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull has-accent-2-background-color has-background" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)"><!-- wp:columns {"align":"wide"} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%"} -->
+		<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"width":"360px","height":"360px","sizeSlug":"large","linkDestination":"none"} -->
+		<figure class="wp-block-image size-large"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/podcast-hero.webp" alt="<?php echo esc_attr_x( 'Picture of a person', 'Podcast Hero Image', 'twentytwentyfive' ); ?>" width="360" height="360"/></figure>
+		<!-- /wp:image -->
+		</div>
+
+		<!-- /wp:column -->
+
+		<!-- wp:column {"verticalAlignment":"center","width":"60%"} -->
+		<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:60%"><!-- wp:heading {"level":1,"fontSize":"xxx-large"} -->
+			<h1 class="wp-block-heading has-xxx-large-font-size">The Stories® Podcast</h1>
```

Comment:
The heading order is incorrect.

The H1 on the page is the page title (post title block or the archive title block), so this should be at least an H2.
The heading levels must not be skipped, an H1 or H2 must not be followed by an H6, it should be the level immediately after.

Reply:
Hey @carolinan thanks also for the feedback here

Considering the figma is a H2 to a H6 how would you best then approach this?

Reply:
The same font size, weight, etc can be used, but the heading levels need to be updated.

I looked at the other patterns that are already merged, but it did give much guidance, its mixed.
Start with H3 followed by H4.
It will be easier to adjust once the patterns are all in and can be combined to different layouts together.

Reply:
Okalie dokalie!! thanks for the guidance here Carolina ill make those changes over the weekend